new.css

```diff
@@ -0,0 +1,316 @@
x #FFF , 1747px 971px #FFF , 895px 1962px #FFF , 730px 1541px #FFF , 1671px 1775px #FFF , 1915px 1108px #FFF , 1554px 1387px #FFF , 1220px 447px #FFF , 1612px 495px #FFF , 1857px 467px #FFF , 1533px 1876px #FFF , 1385px 380px #FFF , 61px 948px #FFF , 800px 932px #FFF , 706px 844px #FFF , 1709px 752px #FFF , 1821px 13px #FFF , 262px 1833px #FFF , 314px 373px #FFF , 1436px 1655px #FFF , 280px 1544px #FFF , 234px 829px #FFF , 41px 44px #FFF , 565px 1128px #FFF , 584px 547px #FFF , 1551px 1592px #FFF , 1658px 973px #FFF , 656px 1268px #FFF , 1330px 161px #FFF , 1578px 729px #FFF , 536px 936px #FFF , 675px 981px #FFF , 96px 1393px #FFF , 1293px 1091px #FFF , 842px 1384px #FFF , 1037px 345px #FFF , 1144px 1054px #FFF , 554px 1546px #FFF , 1750px 1106px #FFF , 1977px 310px #FFF , 1103px 1424px #FFF , 355px 1718px #FFF , 1089px 1599px #FFF , 342px 916px #FFF , 1625px 1275px #FFF , 1131px 1323px #FFF , 1396px 151px #FFF , 1002px 1118px #FFF , 722px 1958px #FFF , 666px 1733px #FFF , 1830px 1836px #FFF , 1069px 1384px #FFF , 45px 1337px #FFF , 498px 1338px #FFF , 1241px 1438px #FFF , 1408px 469px #FFF , 1990px 941px #FFF , 1608px 569px #FFF , 1904px 964px #FFF , 1172px 285px #FFF , 1913px 329px #FFF , 1414px 513px #FFF , 1913px 62px #FFF , 604px 1623px #FFF , 292px 766px #FFF , 941px 961px #FFF , 1612px 469px #FFF , 173px 429px #FFF , 1577px 1531px #FFF , 1780px 1218px #FFF , 1221px 1217px #FFF , 1901px 223px #FFF , 540px 664px #FFF , 1454px 1153px #FFF , 1055px 1876px #FFF , 1332px 927px #FFF , 1796px 1823px #FFF , 1110px 1165px #FFF , 1859px 1834px #FFF , 598px 1845px #FFF , 106px 1829px #FFF , 849px 1028px #FFF , 81px 660px #FFF , 902px 207px #FFF , 764px 889px #FFF , 1641px 999px #FFF , 756px 1759px #FFF , 1867px 938px #FFF , 1635px 1578px #FFF , 1920px 714px #FFF , 239px 1628px #FFF , 1512px 177px #FFF , 585px 1821px #FFF , 1043px 513px #FFF , 1282px 1992px #FFF , 212px 1187px #FFF , 1075px 382px #FFF , 769px 592px #FFF , 89px 1625px #FFF , 1550px 751px #FFF , 1383px 1224px #FFF , 1611px 992px #FFF , 748px 675px #FFF , 889px 1809px #FFF , 756px 19px #FFF , 622px 1367px #FFF , 1925px 1387px #FFF , 534px 868px #FFF , 867px 1773px #FFF , 462px 374px #FFF , 1357px 493px #FFF , 1480px 1858px #FFF , 303px 1828px #FFF , 1058px 209px #FFF , 98px 1758px #FFF , 1542px 634px #FFF , 25px 711px #FFF , 214px 1090px #FFF , 894px 281px #FFF , 1524px 1074px #FFF , 830px 1623px #FFF , 1565px 839px #FFF , 482px 1860px #FFF , 1471px 1518px #FFF , 543px 1315px #FFF , 899px 1432px #FFF , 999px 1619px #FFF , 326px 61px #FFF , 733px 871px #FFF , 1811px 1417px #FFF , 177px 1038px #FFF , 590px 414px #FFF , 1581px 802px #FFF , 603px 1457px #FFF , 242px 372px #FFF , 1144px 482px #FFF , 681px 1428px #FFF , 1789px 435px #FFF , 389px 241px #FFF , 1587px 1078px #FFF , 713px 1861px #FFF , 1601px 1344px #FFF , 149px 1935px #FFF , 587px 594px #FFF , 1767px 1872px #FFF , 1530px 1099px #FFF , 1518px 716px #FFF , 1151px 589px #FFF , 1818px 167px #FFF , 223px 570px #FFF , 456px 1441px #FFF;
```

Comment:
understand how this is working and write the explanation by replying to this comment.

Reply:
box-shadow: 1554px 1004px #FFF , 616px 1871px #FFF , 1756px 1174px #FFF
This part makes a shadow of the div where "1554px 1004px" this is the Location of the shadow and "#FFF" is the colour of the shadow N number os box-shadows are used to make it look like stars.

Later animation is used to make the starts move 
animation: animStar 1s linear infinite;
where animStar is the name for the animation 
1s is the time to move it from point a to b
Infinite is the time for how long the animation should work.

@keyframes animStar {
  from {
    transform: translateY(0px);
  }
  to {
    transform: translateY(50px);
  }
}

This is the keyframes for the animation in which the start and the destination of the stars are marked.